### DISCUSSION TITLE
Warm run

### DISCUSSION THREAD

**Discussion by mullerrobert799** [Deltares/Wflow.jl#289] [Q&A] [answered]
In warm run, the new forcing files of next period should start from time = 0 or from where the last run of first forcing file was ended?

**Comment by JoostBuitink** [comment, marked as answer]
You can use the same forcing files when starting a warm model run, by changing the `starttime` and `endtime` in the .toml file. For example, you run Wflow from 2000-2002 to produce states. These states can then be used to start (many) simulations starting from 2003 until 2020. These runs can therefore all be started using the same forcing file, and by only changing the `starttime`, `endtime`, and `reinit` settings.
